Apromore-Custom-Plugins/User-Admin-Portal-Plugin/src/main/java/org/apromore/plugin/portal/useradmin/UserAdminController.java

```diff
@@ -1144,7 +1144,7 @@ public void onClickRetractUser() {
                 nonAssignedUserList.reset();
                 assignedUserList.reset();
             }
-            isGroupDetailDirty = false;
+            isGroupDetailDirty = true;
```

Comment:
Not in the scope of this PR, but I saw the access modifier for isGroupDetailDirty (and many other instance variables) are by default (protected), they are visible to all classes in the same package. Can we set them to private as Java supports this aspect very well (the most restrictive information hiding principle)? It will help avoid potential troubles.

Reply:
Will fix it